jekyll/_cci2/aws.adoc

```diff
@@ -0,0 +1,232 @@
++
+.Configure VM Service
+image::vmprovider.png[]
+
+You can preallocate instances to always be up and running, reducing the time taken for Remote Docker and `machine` executor jobs to start. If preallocation is set, a cron job will cycle through your preallocated instances once per day to prevent them getting into a bad/dead state.
+
+CAUTION: If Docker Layer Caching (DLC) is to be used VM preallocation must be set to `0` – on-demand – for both Remote Docker and `machine` executor.
+
+//<!--Advanced settings to cover here too - make a section in ops guide for this and list of OK instance types for VM provider-->
+
+. If you wish to use AWS Cloudwatch or Datadog for collating metrics for your installation, set this up here. For more infomration see our https://circleci.com/docs/2.0/monitoring/[Monitoring guidance]:
++
+.Metrics
+image::metrics_setup.png[]
+
+. Artifacts persist data after a job is completed, and may be used for longer-term storage of your build process outputs. By default, CircleCI Server only allows approved types to be served. This is to protect users from uploading, and potentially executing malicious content. The **Artifacts** setting allows you to override this protection. For more information on safe/unsafe types see our https://circleci.com/docs/2.0/build-artifacts/[Build Artifacts guidance].
+
+. After agreeing to the License Agreement and saving your settings, select Restart Now from the popup. You will then be redirected to start CircleCI and view the Management Console Dashboard. It will take a few minutes to download all of the necessary Docker containers.
+
+NOTE: If the Management Console reports `Failure reported from operator: no such image` click Start again and it should continue.
+
+== Validate Your Installation
+
+. When the application is started, select Open to launch CircleCI in your browser, and sign up/log in to your CircleCI installation and start running 2.0 builds! You will become the Administrator at this point as you are the first person to sign in. Have a look at our https://circleci.com/docs/2.0/getting-started/#section=getting-started[Getting Started] guide to start adding projects.
+//<!--add info on making users administrators etc. to user management section of ops guide and put a link here-->
++
+.Start CircleCI from your Dashboard
+image::dashboard.png[]
+
+. After build containers have started and images have been downloaded, the first build should begin immediately. If there are no updates after around **15 minutes**, and you have clicked the Refresh button, contact https://support.circleci.com/hc/en-us[CircleCI support] for assistance.
+
+. You can use https://github.com/circleci/realitycheck[our realitycheck repo] to check basic CircleCI functionality.
+//<!--Is this what we hope they will do? Could use stronger language. Also would be good to have some help for what to do if jobs fail, or do we just want people to tell us so we can help?-->
```

Comment:
Yes, we want them to run reality check, feel free to make this stronger.

Reply:
Great, will do.